dirs.go

```diff
@@ -1,34 +1,31 @@
 package main
 
 import (
-	"errors"
 	"io/fs"
 	"log"
 	"os"
 	"path/filepath"
 )
 
 type Directories struct {
-	Cache  string
-	Config string
-	Cwd    string
-	Data   string
-	Pfx    string
-	Log    string
+	Cache     string
+	Config    string
+	Data      string
+	Downloads string
+	Logs      string
+	Prefix    string
+	Versions  string
 }
 
 var (
-	Dirs                  = defDirs()
-	DirMode   fs.FileMode = 0755
-	DirROMode fs.FileMode = 0664
-	FileMode  fs.FileMode = 0644
+	Dirs                = GetDirectories()
+	DirMode fs.FileMode = 0o755
```

Comment:
typo?

Reply:
Go has these for octal style literals, and this change was done by the `gofumpt` linter.